src/main/java/Attendance.java

```diff
@@ -0,0 +1,21 @@
+
+// Attendance class representing an attendance, with fields date, classSession
+// and student
+public class Attendance {
+
+
+    public Attendance(String date, Student students) {
+        this.date = date;
+        this.students = students;
+    }
+
+    String date;
+    Student students;
```

Comment:
Shouldn't this be an ArrayList ? Right now we would have to add students one by one and keep repeating the date. I think a better design would be to have a list of Students for a specific date -> for a specific event.

Reply:
done